### ISSUE TITLE
Commented-Out Training Data Extraction in create_dataset.py

### ISSUE THREAD

**Issue by GuangtaoLyu** [Luke-Luo1/POPDG#9]
While following the README to process the data, I noticed that the training dataset extraction is commented out in create_dataset.py:
# baseline_extract("train/wavs_sliced", "train/baseline_feats")
baseline_extract(test_wavs, "test/baseline_feats")

However, the EDGE implementation includes both training and test data extraction:
baseline_extract("train/wavs_sliced", "train/baseline_feats")
baseline_extract("test/wavs_sliced", "test/baseline_feats")

My Questions:
Is there a specific reason for commenting out the training data extraction in the current implementation?
Should I uncomment it when training the model, or is there an alternative way to generate training features?
Additionally, why does the commented-out version use test_wavs while the EDGE version uses a hardcoded path "test/wavs_sliced"?
Could you clarify the intended workflow here? I want to ensure I'm processing the data correctly for both training and testing phases.
Thank you!

**Comment by Luke-Luo1** [comment]
In our experiments, we primarily used high-dimensional Jukebox features (4800-dim), which showed better performance than the 35-dim baseline features extracted using Librosa. Therefore, we removed the baseline feature extraction for both the training and test sets (train/wavs_sliced and test/wavs_sliced) from the main workflow.

However, for testing, we still need audio baseline features (e.g., for beat alignment score testing), so we retained baseline extraction for the test set (test_wavs). This is why you see test_wavs used instead of a hardcoded "test/wavs_sliced" path, which had already been removed.

Best regards.

**Comment by GuangtaoLyu** [comment]
thank you.&nbsp;












吕光涛(GuangtaoLyu)
***@***.***






---Original---
From: "Zhenye ***@***.***&gt;
Date: Mon, Jun 23, 2025 11:41 AM
To: ***@***.***&gt;;
Cc: "Guangtao Lyu ( 吕光涛 ***@***.******@***.***&gt;;
Subject: Re: [Luke-Luo1/POPDG] Commented-Out Training Data Extraction increate_dataset.py (Issue #9)


Luke-Luo1 left a comment (Luke-Luo1/POPDG#9)
 
In our experiments, we primarily used high-dimensional Jukebox features (4800-dim), which showed better performance than the 35-dim baseline features extracted using Librosa. Therefore, we removed the baseline feature extraction for both the training and test sets (train/wavs_sliced and test/wavs_sliced) from the main workflow.
 
However, for testing, we still need audio baseline features (e.g., for beat alignment score testing), so we retained baseline extraction for the test set (test_wavs). This is why you see test_wavs used instead of a hardcoded "test/wavs_sliced" path, which had already been removed.
 
Best regards.
